### PR TITLE
🔖(chore) bump version to 2.5.0

### PR DESCRIPTION
**Pull request by jbpenrath**
Release Richie 2.5.0

### Added

- Add support for language fallback on "get_related_blogposts"
- Add portuguese translations

### Fixed

- Avoid KeyError when an object is indexed with no titles
- Add missing rel noopener noreferer on target blank links
- Order blogposts by descending publication date within
  (person|category)_detail template
- Fix filter pane quality issues on handheld devices
- Fix small quality issues on the course search filters modal
- Fix initializeAccordions compatibility issue

### Changed

- Clean built frontend files before each build
- Remove use of regexp group for JS_COURSE_REGEX setting

### CODE REVIEW

**Review by madmatah — APPROVED** [review]

**Review by lunika — APPROVED** [review]

**Review by kernicPanel — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,12 +8,14 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unrealeased]
 
-## Added
+## [2.5.0] - 2021-04-15
```
Comment (sampaccoud) [inline CHANGELOG.md:11]:
```suggestion
## [2.5.0] - 2021-04-21
```

File: website/versions.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
 [
+  "2.5.0",
```
Comment (sampaccoud) [inline website/versions.json:2]: I think you need to regenerate the website because I modified 2 pages in a commit on which you rebased this PR...

Comment (sampaccoud) [inline website/versions.json:2]: Not sure why but you are still missing these changes: https://github.com/openfun/richie/pull/1343/files#diff-5348aab53dc03b67a95ede93b246aa31f9e2a642e74b95c314025e0c4f8e0489R201

Comment (jbpenrath) [reply website/versions.json:2]: This is dynamic content, nothing is generated from it. But if I start website locally, I see these changes.

**Review by sampaccoud — APPROVED** [review]

**Pull request merged**